### PR TITLE
update http chapters

### PR DESCRIPTION
**Pull request by eeliang**
### Which issues does this fix?
https://github.com/workato/docs/issues/954

### PR submission checklist
- [x] Optimised every new image
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to SUMMARY.md

### CODE REVIEW

**Comment by eeliang** [comment]
merge with: https://github.com/workato/workato/pull/22650

File: SUMMARY.md [hunk 1]
```diff
@@ -658,11 +658,9 @@
   * [Registering new client](developing-connectors/registering-new-client.md)
   * [HTTP vs SDK](developing-connectors/http-vs-sdk.md)
   * [HTTP connector V2](developing-connectors/http-v2.md)
```
Comment (eeshansim) [inline SUMMARY.md]: We can remove V2 from the label

File: SUMMARY.md [hunk 2]
```diff
@@ -658,11 +658,9 @@
...
-    * [Connection setup](developing-connectors/http-v2.md#setup-your-http-connection)
-    * [Example: Eventbrite OAuth2 connection](developing-connectors/http-v2.md#example---connecting-to-eventbrite-via-oauth2)
-    * [Webhook triggers](developing-connectors/http.md#building-webhook-triggers)
-    * [Example: New Eventbrite order](developing-connectors/http.md#example---building-a-new-order-trigger-in-eventbrite)
-    * [HTTP actions](developing-connectors/http-v2.md#building-a-workato-action-example---create-venue-in-eventbrite)
+    * [Connection setup](developing-connectors/http/connection-setup.md)
+    * [Building a HTTP trigger](developing-connectors/http/building-http-trigger.md)
+    * [Building a HTTP action](developing-connectors/http/building-http-action.md)
   * [HTTP connector V1 (deprecated)](developing-connectors/http.md)
```
Comment (eeshansim) [inline SUMMARY.md]: remove v1, just say deprecated

File: developing-connectors/http/building-http-action.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+For some endpoints, Workato recommends a sample request to send, which you can either choose to customize further or remove entirely to build it from scratch.
+
+In our example, Workato recommends a sample request body for Eventbrite's create venue endpoint, which we'll choose to use.
+
+![Step 2: RecipeIQ recommendations for HTTP configuration](/assets/images/developing-connectors/http/step2-recipeiq-recommendation.png)
+*Step 2: RecipeIQ recommendations for HTTP configuration*
+
+For endpoints which Workato has not seen before, or if you chose to remove RecipeIQ recommendations, you'll see the following screen.
+
+![Step 2: Configure and send sample request](/assets/images/developing-connectors/http/step2-unconfigured.png)
```
Comment (eeshansim) [inline developing-connectors/http/building-http-action.md]: image

This looks odd with the repeated intercom bubbles. Narrow the screenshot to exclude them.

**Pull request merged**